File: Array/EasyArrayQuestions/Remove Duplicates in-place from Sorted Array.py
arr1 = [1,1,2,2,2,3,3]
removedArr = [arr1[0]]
for i in range(1, len(arr1)):
    if arr1[i] != arr1[i-1]:
        removedArr.append(arr1[i])

print(removedArr)

File: Array/EasyArrayQuestions/Remove Duplicates in-place from Sorted Array.js
let arr1 = [1,1,2,2,2,3,3]
let removedArr=[]
removedArr.push(arr1[0])
for(let i = 1; i<arr1.length; i++){
    if(arr1[i] !== arr1[i-1]){
        removedArr.push(arr1[i])
    }
}

console.log(removedArr)

